nix/latex-documents.nix

```diff
@@ -0,0 +1,146 @@
+{ repoRoot, inputs, pkgs, lib, ... }:
+
+let
+
+  latex-documents = lib.flip lib.mapAttrValues nix.plutus.build-latex-doc {
```

Comment:
If this is supposed to be readable by non-nix experts, this sort of thing makes it harder. I'd just write `build-latex-doc` at each use site, it's not that bad